std/range/package.d

```diff
@@ -4833,14 +4838,15 @@ if (isIntegral!(CommonType!(B, E)) || isPointer!(CommonType!(B, E)))
             return cast(inout Result)Result(cast(Value)(current + lower),
                                             cast(Value)(pastLast - (length - upper)));
         }
-        @property IndexType length() const
+        @property size_t length() const
         {
-            return unsigned(pastLast - current);
+            return cast(size_t)unsigned(pastLast - current);
```

Comment:
Also here. The `unsigned` is redundant. The cast is already forcing the issue. `unsigned` does make sense when comparing again `size_t.max`, since the comparison might be done with a type larger than `size_t`, but when casting, it just results in two casts instead of one.